README.md

```diff
@@ -18,19 +18,19 @@ You can find the content description, speakers and trainers at:
-| Elective F |Starting, Growing, and Stewarding your IPFS Community              | @nukemandan @kk3wong @stefanhans     | Let's get Meta! A workshop on effective workshops! And how to foster a community                                                                | [Course Repo](https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/ELECTIVE_COURSE_F), Slides ([Google Slides](https://docs.google.com/presentation/d/1OLmSfOMjffl7z2ODxHaBcQBmyVEGdt8O7h3bxYv_VVI/edit?usp=sharing))| soon™ |
-| Elective G |Managing Datasets with QRI                                         | @b5                                  | An open source tool for versioned, scriptable, exportable, collaborative datasets | [Course Repo](https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/ELECTIVE_COURSE_G) | soon™ |
-| Elective H |Making the Browser a True User Agent                               | @autonome @Gozala @soapdog           | We’ll show how browsers act as a value filter on our world, teach some approaches to making the browser truly be an agent that represents you, and break into groups to start building our own true user agents.| [Course Repo](https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/ELECTIVE_COURSE_H), Slides ([HackMD](https://hackmd.io/@dietrich/rkrZEKFRV#/)) | soon™ |
+| Elective A | Identity on the DWeb                                               | @satazor @paulobmarcos @joaosantos15 | Understand the state of decentralized identity on the DWeb and current efforts on creating open and interoperable models to solve self-sovereign identities. | [Course Repo](https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/ELECTIVE_COURSE_A), Slides ([Google Slides](https://docs.google.com/presentation/d/1HbydOI0w-T_FY23zCACAyHmzDq1ZvyG2tklpPSm6OQQ))| [Video](https://youtu.be/EsLqjFEN-MM)|
+| Elective B |Managing pinsets with IPFS Cluster                                 | @hsanjuan @lanzafame @meiqimichelle  | A workshop on operating and using IPFS Cluster to orchestrate a common pinset among multiple IPFS nodes | [Course Repo](https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/ELECTIVE_COURSE_B), Slides ([PDF](https://github.com/ipfs/camp/blob/master/CORE_AND_ELECTIVE_COURSES/ELECTIVE_COURSE_B/slides.pdf)) | [Video](https://youtu.be/wmxaVrAFfeE) |
+| Elective C |TBA (But will have cats with lasers...)                            | @gorhgorhi                           |                | [Course Repo](https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/ELECTIVE_COURSE_C) | [Video](https://youtu.be/kVzHE4MRPOM) |
+| Elective D |Building DApps with Textile, the iCloud for the DWeb               | @balupton @carsonfarmer @andrewxhill |                | [Course Repo](https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/ELECTIVE_COURSE_D) | [Video](https://youtu.be/fLUq0RkiTBA) |
```

Comment:
This links to the wrong course, should be [Video](https://youtu.be/4r7_lOyv4W8)

Reply:
fixed the textile one, thanks!